citizenvoice/apiapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,179 @@
+# Generated by Django 4.0.6 on 2022-08-12 08:03
```

Comment:
Reformating was applied to all files, including the ones generated by DJango. That was OK, but a bit unnecessary, files like this are not meant to be modified/read by a developer.